### PR TITLE
Adds @KorPhaeron's artwork to the memorial

### PR DESCRIPTION
**Pull request by Cyberboss**
I thought it looked pretty.

unknown spaceman

Examining the memorial will open a window containing the .png

### CODE REVIEW

**Review by duncathan — COMMENTED** [review]
While you're in here could you please update the comment? "This memorial has been designed for him and any future coders to perish."
I feel like this should instead say "This memorial has been designed for him and any future coders who perish."

**Comment by lzimann** [comment]
annoying

**Comment by epochayur** [comment]
;_;7

**Comment by DoctoreMad** [comment]
Is it bad I always thought it was a lizard

**Comment by KorPhaeron** [comment]
Is there a way to do this without putting the art out under our dumb license that would let anyone use and even sell the work?

**Comment by Arianya** [comment]
Not an expert, but can't you simply store the .png in its own folder, with its separate license stored there?

**Comment by Cyberboss** [comment]
Or I could just make it a config

**Comment by Iamgoofball** [comment]
yeah cyberboss can config it

**Comment by Blute1** [comment]
real talk though that picture is awesome

**Comment by ShadowDeath6** [comment]
accidentally right clicks statue, lags out

can it be like on attackhand or something

**Comment by optimumtact** [comment]
wait aminute, you didn't even get permission?

lol dummy

File: code/game/objects/structures/memorial.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@ but we are all contributors together.
 
 Goodbye, man. We'll miss you.
 
-This memorial has been designed for him and any future coders to perish.
```
Comment (Qustinnus) [inline code/game/objects/structures/memorial.dm]: top kek

Comment (Qustinnus) [reply code/game/objects/structures/memorial.dm]: keep this in i've been laughing for 2 minutes now

**Comment by optimumtact** [comment]
Also the answer to your question kor is no

File: code/game/objects/structures/memorial.dm (hunk 1 above)
Comment (MoreRobustThanYou) [reply code/game/objects/structures/memorial.dm]: fucking insensitive...

**Comment by alexkar598** [comment]
All assets including icons and sound are under a Creative Commons 3.0 BY-SA license unless otherwise indicated.

You should be able to put an other lisence than cc by-sa

**Comment by alexkar598** [comment]
Found that in README.md btw

**Comment by Bawhoppen** [comment]
Changelog?

**Comment by ike709** [comment]
Creative Commons strikes again.

**Pull request closed without merging**